src/app/components/OptimizelyPageMetrics/index.tsx

```diff
@@ -1,16 +1,64 @@
-import React, { useContext } from 'react';
+import React, { useState, useContext, useEffect } from 'react';
+import { OptimizelyContext } from '@optimizely/react-sdk';
 import { RequestContext } from '#contexts/RequestContext';
+import { PageTypes } from '#app/models/types/global';
 import PageCompleteTracking from './PageCompleteTracking';
 import ScrollDepthTracking from './ScrollDepthTracking';
 import PageViewTracking from './PageViewTracking';
+import experimentsForPageMetrics from './experimentsForPageMetrics';
+
+type Props = {
+  trackPageView?: boolean;
+  trackPageDepth?: boolean;
+  trackPageComplete?: boolean;
+  pageType: PageTypes;
```

Comment:
nitpick: move mandatory props to the top
```suggestion
  pageType: PageTypes;
  trackPageView?: boolean;
  trackPageDepth?: boolean;
  trackPageComplete?: boolean;
```

Reply:
No longer applicable